ethpm_types/base.py

```diff
@@ -17,8 +17,8 @@ def json(self, *args, **kwargs) -> str:
         # NOTE: When serializing to IPFS, the canonical representation must be repeatable
 
         # EIP-2678: minified representation (at least by default)
-        if "indent" not in kwargs:
-            kwargs["indent"] = (",", ":")
+        if "separators" not in kwargs:
```

Comment:
Note: `None` is the default value for the `indent=` kwarg so we don't _need_ to specify it